pkg/openthread/contrib/netdev/openthread_netdev.c

```diff
@@ -39,27 +39,38 @@ static msg_t _queue[OPENTHREAD_QUEUE_LEN];
 static kernel_pid_t _pid;
 static otInstance *sInstance;
 
-void ot_exec_job(OT_JOB (*job)(otInstance *, void *), void *context)
-{
-    ot_job_t _job;
+/**
+ * @name    Default configuration for OpenThread network
+ * @{
+ */
+#ifndef OPENTHREAD_PANID
+#define OPENTHREAD_PANID            0x1234
+#endif
+#ifndef OPENTHREAD_CHANNEL
+#define OPENTHREAD_CHANNEL          (26U)
+#endif
+/** @} */
 
-    _job.function = job;
-    _job.context = context;
+uint8_t ot_call_command(char* name, void *arg, void* answer) {
+    ot_job_t job;
+
+    job.mName = name;
```

Comment:
since job is defined in RIOT contrib code, use attribute that matches the RIOT code style:
job.command, job.arg, job.answer